### PR TITLE
Feature/3539  manage membership for on request workspace

### PR DESCRIPTION
**Pull request by asf-algoo**
related to #3539

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been deemed not relevant

### CODE REVIEW

File: backend/tracim_backend/lib/core/content.py [hunk 1]
```diff
@@ -324,7 +324,7 @@ def _revisions_base_query(self, workspace: Workspace = None) -> Query:
     def get_base_query(self, workspace: typing.Optional[Workspace]) -> Query:
         return self._base_query(workspace)
 
-    # TODO - G.M - 2018-07-17 - [Cleanup] Drop this method if unneeded
+    # TODO - G.M - 2020-09-29 - [Cleanup] Should probably be dropped, see issue #704
     # def get_child_folders(self, parent: Content=None, workspace: Workspace=None, filter_by_allowed_content_types: list=[], removed_item_ids: list=[], allowed_node_types=None) -> typing.List[Content]:
     #     """
     #     This method returns child items (folders or items) for left bar treeview.
```
Comment (raphj) [inline backend/tracim_backend/lib/core/content.py:330]: Let's just drop this code commented since 2018, what do you think?

**Review by raphj — APPROVED** [review]

File: backend/tracim_backend/lib/core/content.py (hunk 1 above)
Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py:330]: i do prefer we clean all at the same time see #704

Comment (inkhey) [reply backend/tracim_backend/lib/core/content.py:330]: remove it in another non-related PR make thing a bit more complicated to follow.

**Comment by inkhey** [comment]
@raphj 
I do suggest no manual test on this yet as there is not current direct impact on current frontend (so the only way to test is by using directly api) and specification will probably change, see #3662.

**Comment by raphj** [comment]
ok, let's merge this without manual tests

**Pull request merged**